dev-util/dwdiff/dwdiff-2.1.3.ebuild

```diff
@@ -41,7 +29,3 @@ src_configure() {
 src_compile() {
 	emake CC="$(tc-getCC)" CFLAGS="${CFLAGS}"
 }
-
-src_install() {
-	emake prefix="${D}/usr" docdir="${D}/usr/share/doc/${PF}" install
```

Comment:
so this is all fixed in the bump?

Reply:
Yup, installed tree remains the same without this `src_install`